docs/make.jl

```diff
@@ -0,0 +1,13 @@
+using Documenter, Interpolations
+makedocs(
+sitename="Interpolations.jl",
+modules=[Interpolations],
+pages=["Home" => "index.md",
+        "General usage" => "interpolations.md",
+        "Interpolation algorithms" => "control.md",
+        "Extrapolation" => "extrapolation.md",
+        "Convenience Construcors" => "convenience-construction.md",
+        "Library" => "api.md"]
+)
+
+deploydocs(repo="github.com/felixcremer/Interpolations.jl")
```

Comment:
I've done the necessary key setup for JuliaMath/Interpolations.jl, so you should be able to change this to point to this repo.

Reply:
Changed. The documentation will only be built when the changes are merged, because the docs deployment is pointing on the master branch.